CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 3.25)
 project(PPN-microbench)
-set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD 20)

```

Comment:
Setting the C++ standard this way is considered bad practice as it will also set it for all the dependencies that will be compiled alongside your project. Instead, use compile features on the relevant targets, e.g.:
```cmake
target_compile_features(PPN-microbench PRIVATE cxx_std_20)
```